pom.xml

```diff
@@ -36,7 +40,7 @@ limitations under the License.
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <slf4j.version>1.7.7</slf4j.version>
+    <slf4j.version>1.7.21</slf4j.version>
```

Comment:
Please, let's stick with 1.7.7 all around. While we could change this at one point, we currently enforce versions to be identical in downstream projects.